doc/doxygen/src/Configuration.dox

```diff
@@ -0,0 +1,174 @@
+/*!
+ * @page runtime_configuration Configuring OpenAssetIO
+ *
+ * This page covers how to configure OpenAssetIO for a specific
+ * end-user working environment.
+ *
+ * @section runtime_configuration_overview Overview
+ *
+ * @startuml
+ * node Host {
+ *   package OpenAssetIO {
+ *     collections ManagerPlugin
+ *   }
+ * }
+ *
+ * cloud "Asset Management System" as ams
+ *
+ * ManagerPlugin -right-> ams
+ * ams -left-> ManagerPlugin
+ * @enduml
+ *
+ * An OpenAssetIO enabled tool or application is known as a @ref host "Host"
+ * (as it "hosts" the API, tenuous - but naming things is hard). Through
+ * the use of a plugin system, OpenAssetIO allows the host to
+ * communicate with one or more @ref asset_management_system "Asset Management Systems",
+ * called @ref manager "Managers".
+ *
+ * This page covers hows to configure OpenAssetIO so that the hosts you
+ * use (e.g. @ref DCC "DCC tools", applications and other scripts) use a
+ * specific manager of your choice to @ref resolve and potentially @ref
+ * publish data.
+ *
+ * OpenAssetIO itself is usually built into any host that supports it,
+ * this means you don't normally need do to anything to ensure the API
+ * itself is present. The most common tasks in configuration are:
+ *
+ * - Installing manager plugins and making them available to a host.
+ * - Configuring which manager a host should use.
+ *
+ *
+ * @section installing_manager_plugins Installing Manager plugins
+ *
+ * @note To aid problem solving, the plugin system logs debug messages
+ * when it searches for, and loads plugins.
+ *
+ * Presently, OpenAssetIO only supports Python based plugins.
+ * These can be installed in one of two ways.
+ *
+ * @subsection installing_manager_plugins_pip Using pip
+ *
+ * Most Python-based plugins should support what is known as
+ * <a href="https://packaging.python.org/en/latest/specifications/entry-points/" target="_blank">"entry point" based loading</a>.
+ * This means they can be installed using the package installer for
+ * Python, and will then be automatically detected.
+ *
+ * These plugins are usually published to
+ * <a href="https://pypi.org" target="_blank">PyPI</a>, and so are
+ * easy to install. Simply use the `pip` module of the Python
+ * installation you wish to make the plugin available to:
+ *
+ * @code{.sh}
+ * python -m pip install <manager-plugin-package>
+ * @endcode
+ *
+ * If the plugin is not published, you may be able to install it from a
+ * local checkout of its source code:
+ *
+ * @code{.sh}
+ * python -m pip install <path/to/src>
+ * @endcode
+ *
+ * @warning Many hosts use their own embedded Python interpreter. You
+ * may need to consult the relevant documentation to determine its
+ * location, version, `sys.path` etc. and adjust the invocation
+ * of `python` accordingly.
```

Comment:
[Consider]
```suggestion
 * location, version, `sys.path` etc. and adjust the invocation
 * of `python` accordingly, so that the plugin is installed to the
 * correct python instance.
```

I think that's the reason this warning exists right, to make sure folks install the libs to the correct python version? If so, I think the extra explanation reduces ambiguity.

Reply:
https://github.com/OpenAssetIO/OpenAssetIO/pull/860/commits/3ebc71bdda17d53e4ed89bec330ae8a93f9c1846